cwms-data-api/src/main/java/cwms/cda/api/RatingController.java

```diff
@@ -369,23 +367,24 @@ public void getOne(@NotNull Context ctx, @NotNull String rating) {
                 endInstant = DateUtils.parseUserDate(end, timezone).toInstant();
             }
 
+            String officeId = ctx.queryParam(OFFICE);
+
             RatingSet.DatabaseLoadMethod method = ctx.queryParamAsClass(METHOD,
                     RatingSet.DatabaseLoadMethod.class)
                     .getOrDefault(RatingSet.DatabaseLoadMethod.EAGER);
 
-            String body = getRatingSetString(ctx, method, officeId, rating, beginInstant, endInstant);
+            String body = getRatingSetString(ctx, method, officeId, rating, beginInstant, endInstant, false);
```

Comment:
The changes to this controller should be gone right? the latest option is moved to `RatingLatestController`

Reply:
Correct, I moved any remnants to the RatingLatestController and cleaned up the original controller